pinot-controller/src/test/java/org/apache/pinot/controller/api/PinotSchemaRestletResourceTest.java

```diff
@@ -53,6 +55,36 @@ public void testBadContentType() {
     Assert.fail("Should have caught an exception");
   }
 
+  @Test
+  public void testPostJson() {
+    String schemaString = "{\n" + "  \"schemaName\" : \"transcript\",\n" + "  \"dimensionFieldSpecs\" : [ {\n"
+        + "    \"name\" : \"studentID\",\n" + "    \"dataType\" : \"STRING\"\n" + "  }, {\n"
+        + "    \"name\" : \"firstName\",\n" + "    \"dataType\" : \"STRING\"\n" + "  }, {\n"
+        + "    \"name\" : \"lastName\",\n" + "    \"dataType\" : \"STRING\"\n" + "  }, {\n"
+        + "    \"name\" : \"gender\",\n" + "    \"dataType\" : \"STRING\"\n" + "  }, {\n"
+        + "    \"name\" : \"subject\",\n" + "    \"dataType\" : \"STRING\"\n" + "  } ],\n"
+        + "  \"metricFieldSpecs\" : [ {\n" + "    \"name\" : \"score\",\n" + "    \"dataType\" : \"FLOAT\"\n"
+        + "  } ]}";
+    try {
+      Map<String, String> header = new HashMap<>();
+      sendPostRequest(_controllerRequestURLBuilder.forSchemaCreate(), schemaString, header);
+    } catch (IOException e) {
+      e.printStackTrace();
```

Comment:
Use logger?

Reply:
removed this, since we know what the exception is.